### ISSUE TITLE
Monkeypox Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#60]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Monkeypox Detection</br>
:red_circle: **Aim** : Create a image processing model using the DL methods by which we can identify the images which are affected by Monkeypox.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/arafathussain/monkeypox-skin-image-dataset-2022</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by AKSHAYRAOCH** [comment]
@abhisheks008 I would like to work on this issue
Full name: Chennamaneni Akshay Rao
GitHub Profile Link:https://github.com/AKSHAYRAOCH
Email ID:chakshayrao@gmail.com
Participant ID (if applicable):
**Approach for this Project**: I will try using CNN and classifications algorithms like decision trees or random forest and SVM.
What is your participant role?: contributor

**Comment by abhisheks008** [comment]
It's an image processing kind of project, you have been given a set of images as per the dataset, and you need to detect the diseased one from those. I think you need to think about the deep learning and neural networks oriented approach @AKSHAYRAOCH

**Comment by abhisheks008** [comment]
Issue assigned to you @AKSHAYRAOCH

**Comment by 072arushi** [comment]
@AKSHAYRAOCH You have been unassigned.

**Comment by omkar3602** [comment]
Full name: Omkar Jahagirdar
Github Profile Link: https://github.com/omkar3602
Email ID: omkar3602@gmail.com
Approach: I would like to work on this issue. I will be using CNN models with different architectures. I will also be exploring transfer learning(pre-trained models) if i don't get a good accuracy from CNN models.
Participant role: SSOC

**Comment by abhisheks008** [comment]
Issue assigned to you @omkar3602

**Comment by abhisheks008** [comment]
@omkar3602 use this instead of the previous one, https://www.kaggle.com/datasets/nafin59/monkeypox-skin-lesion-dataset
